### ISSUE TITLE
MSFT-47[Login] "Forgot your password?" hyperlink foreground and background colors won't meet WCAG 2 AA contrast ratio thresholds

### ISSUE THREAD

**Issue by veshu** [veshu/testjiraimport#118]
**Steps to Reproduce**
1. Go to Go to https://accounts.topcoder.com/member
2. Check the "Forgot your password?" hyperlink

**Expected Result**
Expected contrast ratio of 4.5:1

- Passes AA, if you change background color to #2C2C2C. New contrast ratio: 4.53
- Passes AA, if you change text color to #0079CE. New contrast ratio: 4.54

Some people with low vision experience low contrast, meaning that there aren't very many bright or dark areas. Everything tends to appear about the same brightness, which makes it hard to distinguish outlines, borders, edges, and details. Text that is too close in luminance (brightness) to the background can be hard to read.

**Actual Result**
Element has insufficient color contrast of 3.08 (foreground color: #0096ff, background color: #ffffff, font size: 9.0pt, font weight: normal)

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome
**WCAG Category**: 1. Perceivable
**WCAG Level**: AA
**WCAG Success Criterion**: 1.4.3 Contrast (minimum)

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=true&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F26
